src/Examples.Utils/CIFARReader .cs

```diff
@@ -108,11 +108,11 @@ private int ReadSingleFile(string path, int batch_size, bool shuffle, Device dev
         public IEnumerable<(Tensor, Tensor)> Data()
         {
             for (var i = 0; i < data.Count; i++) {
-                yield return (data[i], labels[i]);
-
+                var tfdata = data[i];
                 foreach (var tfrm in _transforms) {
-                    yield return (tfrm.forward(data[i]), labels[i]);
```

Comment:
The idea of this example code was actually to separately apply all transforms that are passed in, not pipeline them together like the modified code does.